dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/task/AbstractJdbcTask.java

```diff
@@ -89,24 +90,33 @@ protected CSVFormat newCsvFormat(@Nonnull ResultSet rs) throws SQLException {
                 // Can we avoid nesting exceptions here?
                 throw new SQLException(
                         new MetadataDumperUsageException("Fatal Error. ResultSet does not have the expected column count: " + headerClass.getEnumConstants().length,
-                                Arrays.asList("If a custom query has been specified please confirm the selected columns match the following: ",
-                                        StringUtils.join(headerClass.getEnumConstants(), ", "))));
+                            Arrays.asList(
+                                "If a custom query has been specified please confirm the selected columns match the following: ",
+                                StringUtils.join(headerClass.getEnumConstants(), ", "))));
```

Comment:
Similarly to the previous comment: please, revert the formatting-only change or move it to a separate PR.